aarch64-unknown-linux-postgres.json

```diff
@@ -0,0 +1,31 @@
+{
+    "arch": "aarch64",
+    "crt-static-respected": true,
+    "data-layout": "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128",
+    "dynamic-linking": true,
+    "env": "gnu",
+    "executables": true,
+    "features": "+outline-atomics",
+    "has-rpath": true,
+    "has-thread-local": true,
+    "llvm-target": "aarch64-unknown-linux-gnu",
+    "max-atomic-width": 128,
+    "os": "linux",
+    "position-independent-executables": true,
+    "relro-level": "full",
+    "supported-sanitizers": [
+        "address",
+        "cfi",
+        "leak",
+        "memory",
+        "memtag",
+        "thread",
+        "hwaddress"
+    ],
+    "target-family": [
+        "postgres",
+        "unix"
+    ],
+    "target-mcount": "\u0001_mcount",
+    "target-pointer-width": "64"
+}
```

Comment:
```suggestion
}

```

Reply:
Strangely enough my editor kept removing that last empty line.